addons/account_edi_ubl_cii/models/account_edi_common.py

```diff
@@ -111,6 +111,17 @@ def _get_uom_unece_code(self, line):
     # TAXES
     # -------------------------------------------------------------------------
 
+    def _validate_taxes(self, invoice):
+        """ Validate the structure of the tax repartition lines (invalid structure could lead to unexpected results)
+        """
+        for invoice_line in invoice.invoice_line_ids:
+            for tax in invoice_line.tax_ids:
+                try:
+                    tax._validate_repartition_lines()
+                except ValidationError as e:
+                    error_msg = _("Tax '%s' is invalid: ") % tax.name
+                    raise ValidationError(error_msg + e.args[0])
```

Comment:
```suggestion
                    error_msg = _("Tax '%s' is invalid: %s", tax.name, e.args[0])
                    raise ValidationError(error_msg)
```
It's clearer to build the message all at once. Also IMO a comment to explain what e.args[0] returns would be useful as it's not clear until you look/try :)
Ofc if you change it don't forget to update the pot